apps/cbportal/base/CBGrid/0-access Node/0-access Node.wiki

```diff
@@ -1,4 +1,15 @@
+
 {{datatables_use}}
-below are the sessions for $$name if you want to provision a session click [here|/cbgrid/provision?ip=$$ip&node=$$node]
-h2. Sessions for $$name
-{{accesssessions remote:$$ip }}
\ No newline at end of file
+below are the sessions for ${name} if you want to provision a session click [here|/cbgrid/provision?node=$$node]
+h2. Sessions for ${name}
+{{accesssessions id:$$node}}
+
+||Session ID||Username||Remote||Start||End||
+{% for table in tables -%}
+|${table[0]}|${table[1]}|${table[2]}|${table[3]}|${table[4]}|
+{% endfor %}
```

Comment:
wwhat happens when table is emtpy? 

i have in the past written an else with a empty row stating there is no data... otherwise header dissapears or crash dont rememer wll